src/EditorFeatures/Core/ModernCommands/GoToBaseCommandArgs.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Editor.Commanding;
+
+namespace Microsoft.CodeAnalysis.Editor.Commanding.Commands
+{
+    /// <summary>
+    /// Arguments for Go To Implementation.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    internal sealed class GoToBaseCommandArgs : EditorCommandArgs
```

Comment:
As mentioned earlier, this won't be necessary when it's implemented on the editor side.